File: despatch.WEB/app/src/swagger/models/room-image-response.ts
/* tslint:disable */
import { BaseEntityResponse } from './base-entity-response';
import { ImageResponse } from './image-response';
import { RoomResponse } from './room-response';
export interface RoomImageResponse extends BaseEntityResponse{
  image?: ImageResponse;
  imageId?: string;
  room?: RoomResponse;
  roomId?: string;
}

File: despatch.Domain/Models/Default/Response/RoomImageResponse.cs
namespace despatch.Domain.Models.Default.Response
{
    public class RoomImageResponse : BaseEntityResponse
    {
        public string RoomId { get; set; }
        public string ImageId { get; set; }

        public virtual RoomResponse Room { get; set; }
        public virtual ImageResponse Image { get; set; }
    }
}
